tests/browser_test.go

```diff
@@ -153,3 +153,34 @@ func TestBrowserCrashErr(t *testing.T) {
 		newTestBrowser(t, lopts)
 	}, "launching browser: Invalid devtools server port")
 }
+
+func TestBrowserLogTraceID(t *testing.T) {
+	t.Parallel()
+
+	tb := newTestBrowser(t, withLogCache())
+
+	var (
+		traceID    string
+		tracedEvts int
+	)
+
+	tb.logCache.mu.RLock()
+	defer tb.logCache.mu.RUnlock()
+	for _, evt := range tb.logCache.entries {
+		for k, v := range evt.Data {
+			if k == "trace_id" {
+				require.IsType(t, "", v)
+				require.NotEmpty(t, v)
+				tracedEvts++
+				if traceID == "" {
+					traceID, _ = v.(string)
```

Comment:
Is there no way to get the `traceID` from the context, and then compare that against the `traceID` from the log lines?

Reply:
If there isn't a way to retrieve the `traceID` from the context, then should we ensure that the `v.(string)` itself isn't an empty string?

Reply:
Yeah, sure, that seems slightly better :+1:

Reply:
Resolved by 7642c45.